v0/destinations/mp/data/MPEventPropertiesConfig.json

```diff
@@ -112,15 +115,24 @@
     "destKey": "mp_processing_time_ms"
   },
   {
-    "sourceKeys": "properties.appBuildNumber",
+    "sourceKeys": [
+      "properties.appBuildNumber",
+      "context.app.build"
+    ],
     "destKey": "$app_build_number"
   },
   {
-    "sourceKeys": "properties.appVersionString",
+    "sourceKeys": [
+      "properties.appVersionString",
+      "context.app.version"
+    ],
     "destKey": "$app_version_string"
   },
   {
-    "sourceKeys": "properties.brand",
+    "sourceKeys": [
+      "properties.brand",
+      "device.manufacturer"
+    ],
     "destKey": "$brand"
   },
   {
```

Comment:
for `$radio` its written
Radio ($radio)
Current [cellular network communication standard](https://www.objc.io/issues/5-ios7/iOS7-hidden-gems-and-workarounds/#know-your-radio) (3G, 4G, LTE, etc).

We should be able to use carrier info for this